src/components/Model3DViewer/stories/full.md

```diff
@@ -4,10 +4,10 @@
 
 #### Requirements:
 
-To use this component you have to install `@cognite/3d-viewer` package (version 4.1.3):
+To use this component you have to install `@cognite/3d-viewer` package (version 5.0.7):
 
 ```bash
-yarn add @cognite/3d-viewer@4.1.3
+yarn add @cognite/3d-viewer@5.0.7
```

Comment:
Not only yarn add @cognite/3d-viewer@5 ?

Reply:
true, doing